dask_expr/tests/test_rolling.py

```diff
@@ -0,0 +1,83 @@
+import pytest
+
+from dask_expr import from_pandas
+from dask_expr.tests._util import _backend_library, assert_eq
+
+# Set DataFrame backend for this module
+lib = _backend_library()
+
+
+@pytest.fixture
+def pdf():
+    idx = lib.date_range("2000-01-01", periods=12, freq="T")
+    pdf = lib.DataFrame({"foo": range(len(idx))}, index=idx)
+    pdf["bar"] = 1
+    yield pdf
+
+
+@pytest.fixture
+def df(pdf, request):
+    npartitions = getattr(request, "param", 1)
+    if npartitions > 1:
+        pytest.xfail("rolling with npartitions>1 not implemented")
+    yield from_pandas(pdf, npartitions=npartitions)
+
+
+@pytest.mark.parametrize(
+    "api,how_args",
+    [
+        ("count", ()),
+        ("mean", ()),
+        ("sum", ()),
+        ("min", ()),
+        ("max", ()),
+        ("var", ()),
+        ("std", ()),
+        ("median", ()),
+        ("skew", ()),
+        ("quantile", (0.5,)),
+        ("kurt", ()),
+    ],
+)
+@pytest.mark.parametrize("window", (2,))
```

Comment:
can you add 1?

Reply:
Done https://github.com/dask-contrib/dask-expr/pull/405/commits/608f7e8549d07e3933d81bcb2939ace69733495f